CMakeLists.txt

```diff
@@ -86,6 +86,13 @@ include("${CMAKE_CURRENT_SOURCE_DIR}/config/XSDKDefaults.cmake")
 
 # Enable languages.
 enable_language(CXX)
+if (MFEM_USE_CUDA)
+   # MFEM_USE_CUDA requires CMake 3.8 or newer (for direct CUDA support)
+   cmake_minimum_required(VERSION 3.8 FATAL_ERROR)
+   set(CUDA_PROPAGATE_HOST_FLAGS ON)
```

Comment:
This option does not seem to work - looks like it is only used by the old `CUDA` module (`FindCUDA.cmake`).

I'll remove it unless there are any objections.

Reply:
You're right, that may not be useful anymore.